### PR TITLE
Fix startup scripts

### PR DESCRIPTION
**Pull request by root-beer**

### CODE REVIEW

File: scripts/dev_server.sh [hunk 1]
```diff
@@ -8,4 +8,14 @@
 # http://opensource.org/licenses/mit-license.php
 #
 
+# default set SkyHopper path
+if [ -z "$SKYHOPPER_PATH" ]; then
+  SKYHOPPER_PATH="$(cd $(dirname $0); cd ../; pwd)"
+fi
+
+PIDS_PATH="${SKYHOPPER_PATH}/tmp/pids"
```
Comment (Joeper214) [inline scripts/dev_server.sh]: https://github.com/skyarch-networks/skyhopper/blob/fix_startup_scripts/scripts/dev_server.sh#L16-L19

:+1:

**Comment by Joeper214** [comment]
Please fix conflicts. @root-beer . Thanks.

**Comment by pocke** [comment]
I'll review now

**Comment by pocke** [comment]
LGTM!

**Pull request merged**